<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginGenerator</class>
 <widget class="QWidget" name="PluginGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,2,4,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Desktop Entry file - [Resource Folder]/pykrita/[short name].desktop</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plugin Title: (Name in plugin manager)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Short Name: (code name without spaces or special characters used for folder name)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="shortNameEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Short Description: (Shows up in plugin manager)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="shortDescEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Manual - [Resource Folder]/pykrita/[short name]/Manual.html</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Long Description/Manual: (basic HTML format, you can edit it in a text editor later)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="manualTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Setup - [Resource Folder]/pykrita/[short name]/*</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Project Path: (project path will symlink to your pykrita folder)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,0,0">
        <item>
         <widget class="QLabel" name="projectPathLabel">
          <property name="text">
           <string>Select Path...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="projectPathSymlinkChk">
          <property name="text">
           <string>Use Symlink</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="projectPathBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Template:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="templateListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="hotkeysChk">
          <property name="text">
           <string>Generate custom hotkey shortcuts ([Resource Folder]/actions/[short name].action)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="hotkeysBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="setupGitChk">
          <property name="text">
           <string>Setup Git (requires git to be installed)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="setupGitBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="autocompleteChk">
        <property name="text">
         <string>Generate AutoComplete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="doneBtn">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
